lib/ansible/plugins/vars/sops.py

```diff
@@ -0,0 +1,464 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Plugins contributed to Ansible must be GPLv3+ licensed.

Reply:
Okay, I believe I can do that. Since this is heavily borrowed from https://github.com/mozilla/sops/blob/python-sops/sops/__init__.py, fyi @jvehent (the last contributor to that file) that I will be submitting this under the GPLv3+ rather than the MPL v2.